var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -106,6 +100,8 @@ class Hwloc(AutotoolsPackage, CudaPackage, ROCmPackage):
         "oneapi-level-zero", default=False, description="Support Intel OneAPI Level Zero devices"
     )
 
+    depends_on("c", type="build")
+    depends_on("cxx", type="build")
```

Comment:
cxx isn't strictly required, it's only used during examples build, but configure disables it if no C++ compiler is found.